hls4ml/backends/fpga/fpga_backend.py

```diff
@@ -2,14 +2,13 @@
 import math
 import os
 from bisect import bisect_left
-from queue import Queue
 from collections.abc import Iterable
 import re
 
 from hls4ml.backends.backend import Backend
-from hls4ml.model.layers import Layer
-from hls4ml.model.attributes import Attribute
-from hls4ml.model.types import IntegerPrecisionType, FixedPrecisionType, XnorPrecisionType, ExponentPrecisionType
+from hls4ml.model.layers import Layer, Dense, BatchNormalization, Activation, Softmax, Conv1D, Conv2D, SeparableConv1D, SeparableConv2D, Pooling1D, Pooling2D, GlobalPooling1D, GlobalPooling2D, SimpleRNN, LSTM, GRU, Dot, Embedding, GarNet, GarNetStack
```

Comment:
I would prefer doing something like
```python
from hls4ml.model.layers import (
    Layer, Dense, BatchNormalization, Activation, Softmax, Conv1D, Conv2D, SeparableConv1D,
    SeparableConv2D, Pooling1D, Pooling2D, GlobalPooling1D, GlobalPooling2D, SimpleRNN,
    LSTM, GRU, Dot, Embedding, GarNet, GarNetStack)
```
for the long lines. This is what I see in the matplotlib source code, for example:
https://github.com/matplotlib/matplotlib/blob/main/lib/matplotlib/backends/backend_qt.py#L9-L12

(As an aside, VS code tells me that `Layer` is not used explicitly.)

Reply:
Hi @jmitrevs, I wouldn't worry about formatting things like this because we're going to run the `pre-commit` hooks on this PR after the review, which will resolve all of this